src/modules/mc_pos_control/mc_pos_control_params.c

```diff
@@ -395,6 +395,17 @@ PARAM_DEFINE_FLOAT(MPC_MAN_TILT_MAX, 35.0f);
  */
 PARAM_DEFINE_FLOAT(MPC_MAN_Y_MAX, 150.0f);
 
+/**
+ * Manual yaw rate time constant
```

Comment:
```suggestion
 * Manual yaw rate input low pass filter time constant
```
Although it might not be desired in most cases it's not possible to disable the filter with this option.

Reply:
a time constant of zero sets alpha to 1. The filter is hence disabled

Reply:
I'll add some suggestions to make this clear to the user.